src/schemas/Comparison.json

```diff
@@ -60,6 +60,37 @@
       "required": ["comparison", "target"],
       "additionalProperties": false
     },
+    {
```

Comment:
This isn't correct. The "target" field is for referencing another attribute to compare to like so:

```
{ 
    "user.patients": {
       comparison: "includes",
       target: "patient.id"
    }
}
```

An attribute name should never be a UUID. For a UUID (or any other constant), the field to use is `value` rather than `target` and `value` already allows UUIDs.